docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc

```diff
@@ -566,6 +566,11 @@ Do you confirm y/n :y
 ----
 NB: Once the command is executed, the CLI will reload the server and reconnect to it.
 
+This command can also obtain the certificates from the Let's Encrypt certificate authority
+by use of the --lets-encrypt parameter.
+Besides the mentioned workflow the user will be prompted to specify more information
+such as info about account key store and certificate authority account, in order to obtain the certificate from Let's Encrypt.
```

Comment:
I would replace with:
Besides the mentioned workflow the user will be prompted to specify more information (eg: account key store,  certificate authority account) to obtain the certificate from Let's Encrypt.
Seems to sound better. Applies also to other place.

Reply:
@MMarus Can you fix^^?

Reply:
@kabir  fixed :). Sorry I probably overlooked this.